src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementController.java

```diff
@@ -62,7 +63,7 @@ public void invokeHearing(@RequestBody @Valid HearingRequest hearingRequest) {
         @ApiResponse(code = 200, message = "Success (with content)"),
         @ApiResponse(code = 400, message = "Invalid request")
     })
-    public HearingRequest getHearingsRequest(@PathVariable("ccdCaseRef")
+    public HearingsGetResponse getHearingsRequest(@PathVariable("ccdCaseRef")
```

Comment:
why has the response changed? To me this is less clear

Reply:
Dan pointed out that the response wasn't correct. The response should have been a limited selection of fields.  In hindsight, this should have been address in a separate JIRA.